app/senders/card_mail_sender.rb

```diff
@@ -1,23 +1,29 @@
 class CardMailSender
-  def self.card_moved(card, _)
-    return unless card.changed_list? && card.involved_users.present?
+  def self.card_moved(card, originated_by, move_params)
```

Comment:
Unused method argument - `move_params`. If it's necessary, use `_` or `_move_params` as an argument name to indicate that it won't be used.